ansible_collections/arista/avd/roles/eos_designs/defaults/main/main.yml

```diff
@@ -26,7 +26,6 @@ design:
 
 mgmt_vrf_routing: false
 
-mgmt_interface: Management1
```

Comment:
CI error caused because you deleted this, as this knob is used in upgrade_2.x_to_3.x role. But please comment above this knob of it's depreciation if you are not deleting this line. So we can remove in later versions. Else we have to update in the 2.x_to_3.x upgrade role. Whatever is the best option!